uaclient/entitlements/fips.py

```diff
@@ -59,9 +60,7 @@ class FIPSEntitlement(FIPSCommonEntitlement):
     title = "FIPS"
     description = "NIST-certified FIPS modules"
     messaging = {
-        "post_enable": [
-            "FIPS configured and pending, please reboot to make active."
-        ]
+        "post_enable": ["A reboot is required to complete the install"]
```

Comment:
If we have a generic `reboot_on_enable` attribute now, should this messaging also be made generic? Would this perhaps be better implemented as specific to the FIPS classes for now (perhaps by overriding `enable` with a super call?), until we have more than one place it's needed?

Reply:
I think since we are limiting the implementation of reboot_required feature to just fips* entitlements, it makes sense for us the make the generic post_enable messaging common in FIPSCommentEntitlement parent class. Until we have other use-cases up at the RepoEntitlement parent class, I don't think we should promote this generic messaging and handling up to that level yet. I've promoted the generic post_enable messaging to FIPSCommonEntitlement for this branch.